tests/import_test.go

```diff
@@ -1044,6 +1053,13 @@ var _ = Describe("Preallocation", func() {
 			md5, err := f.GetMD5(f.Namespace, pvc, path, md5PrefixSize)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(md5).To(Equal(expectedMD5))
+
+			if !f.IsBlockVolumeStorageClassAvailable() {
```

Comment:
Sure you need this if only here?

Reply:
This is the only test where Block volumes are used.